sonar-scanner-engine/src/main/java/org/sonar/scanner/sensor/DefaultSensorStorage.java

```diff
@@ -361,8 +358,20 @@ public void store(DefaultHighlighting highlighting) {
     if (writer.hasComponentData(FileStructure.Domain.SYNTAX_HIGHLIGHTINGS, componentRef)) {
       throw new UnsupportedOperationException("Trying to save highlighting twice for the same file is not supported: " + inputFile.absolutePath());
     }
+    final ScannerReport.SyntaxHighlightingRule.Builder builder = ScannerReport.SyntaxHighlightingRule.newBuilder();
+    final ScannerReport.TextRange.Builder rangeBuilder = ScannerReport.TextRange.newBuilder();
+
     writer.writeComponentSyntaxHighlighting(componentRef,
-      Iterables.transform(highlighting.getSyntaxHighlightingRuleSet(), new BuildSyntaxHighlighting()));
+      highlighting.getSyntaxHighlightingRuleSet().stream()
+        .map(input -> {
+          builder.setRange(rangeBuilder.setStartLine(input.range().start().line())
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)